config/pmd.xml

```diff
@@ -7,9 +7,40 @@
   <description>
     PMD common ruleset for Checkstyle main/test sourcesets
   </description>
+  <rule ref="category/java/errorprone.xml">
+    <!-- That rule is not practical, no options to allow some magic numbers,
+         we will use our implementation. -->
+    <exclude name="AvoidLiteralsInIfCondition"/>
+    <!-- Too many false positives. -->
+    <exclude name="BeanMembersShouldSerialize"/>
```

Comment:
Is it new rule ? I did not find it in config before your changes.

Reply:
@pbludov , 
strange, rule was in pmd for a while - http://pmd.sourceforge.net/pmd-4.3.0/rules/javabeans.html, we did not excluded it, It might did not work(no violation). 

comment should be "it is not our goal for now keep all Serializable, may be in future"

Reply:
this rule was in old PMD version , but did not make any violation, might be fixed in 6.0.

suppress comment: "it is not our goal to make all Serializable for now"

Reply:
this rule was in old PMD version , but did not make any violation, might be fixed in 6.0.

suppress comment: "it is not our goal to make all Serializable for now"